tests/integration/test_agentd/test_agentd_multi_server/data/configuration_templates/wazuh_conf.yaml

```diff
@@ -0,0 +1,49 @@
+    - tags:
+      - all
+      sections:
+      - section: client
+        elements:
+            - notify_time:
+                value: 10
+            - time-reconnect:
+                value: 60
+            - auto_restart:
+                value: 'yes'
+            - crypto_method:
+                value: 'aes'
+            - server:
+                elements:
+                - address:
+                    value: 127.0.0.1
+                - port:
+                    value: 1514
+                - protocol:
+                    value: PROTOCOL
+                - max_retries:
+                    value: 3
+                - retry_interval:
+                    value: 3
+            - server:
+                elements:
+                - address:
+                    value: 127.0.0.1
+                - port:
+                    value: 1516
+                - protocol:
+                    value: PROTOCOL
+                - max_retries:
+                    value: 3
+                - retry_interval:
+                    value: 3
+            - server:
+                elements:
+                - address:
+                    value: 127.0.0.1
+                - port:
+                    value: 1517
+                - protocol:
+                    value: PROTOCOL
+                - max_retries:
+                    value: 3
+                - retry_interval:
+                    value: 3
```

Comment:
Missing empty line at the end of the file. Check all files.

Reply:
Addressed with https://github.com/wazuh/wazuh/pull/18353/commits/5a1b7778d254f00b85c507ace508492c18df7d0c